posts/business-leaders-and-digital-transformation.md

```diff
@@ -0,0 +1,59 @@
+    * A/B testing - When mapping paths for development, leadership often encounters uncertainty - a fork in the road if you will - where existing data is not sufficient to determine the best path forward. Running experiments with real users allows leaders to gauge customer engagement. This method is critical to reducing ambiguity and in making data driven-decisions. 
+    * Community-driven development - the utilization of open-source software. "Why reinvent the wheel?" There are no extra points in business for building out a solution that already exists. By leveraging open-source software, businesses can benefit on multiple fronts. The first being that the software is tried and tested, with robust documentation. The second being that costs for development can be significantly reduced by leveraging crowd-sourcing for maintenance and feature development. By focusing on differentiating features, rather than the development and maintenance of a custom codebase, companies can move faster and do more, for less.
+
+## Enablers
+1. In a business setting, understanding the strategic advantages of how and where your technology fits into the market is critical. The same principles classically applied to portfolio management can now be applied to software development:
+    * Market attractiveness and strategic positioning - what is the value proposition of the technology being adopted, and how does this position the business for short- and long-term success?
+    * Investment analysis and allocation: When deciding on the what and how and assessing which technology initiative to pursue or between technologies to help meet that goal, what is the cost-benefit of choosing one over the other? In large enterprises, that decision can be costly, especially if it's the wrong one.
+    * Risk assessment - how does a business mitigate risk? Whether it's distributing risk by using multiple computing services, meeting uptime commitments, quickly catching bugs and fixing bugs, or deterring bad actors - maintaining a reputation for dependability and security can make or break companies.
+
+2. Talent and governance - Agility, ownership and accountability are critical to business success. By using smaller teams with clear ownership and accountability, the development of large technology initiatives can be more agile, performance measured more clearly, and problems addressed more quickly.  
+
+## Conclusion
+
+Leveraging technology has become critical across numerous sectors and will inevitably make its way into every industry, but few companies are able to leverage it to their best ability and it can even become a roadblock in reaching their full potential. By ensuring technology becomes a core element of strategy development at the very top, companies will be able to effectively use many more of the tools at their disposal and make smarter, data-driven decisions.
```

Comment:
```suggestion
Leveraging technology has become critical across numerous sectors and will inevitably make its way into every industry, but few companies are able to leverage it to the best of their ability. It can even become a roadblock in reaching their full potential. By ensuring technology becomes a core element of strategy development at the very top, companies will be able to effectively use many more of the tools at their disposal and make smarter, data-driven decisions.
```